### DISCUSSION TITLE
user_project_wrapper.v內找不到lab3 的module

### DISCUSSION THREAD

**Discussion by ghost** [bol-edu/HLS-SOC-Discussions#139] [Q&A]
我在user_project_wrapper.的資料夾內加入我新的design與lab3 的fir.v，並在user_project_wrapper.v中將這些design接線(下圖1)，但會發報error(下圖2)，感覺是wrapper沒有包含到這些.v檔，不知道需要怎麼修正這個錯誤。
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/119398292/f4f46ab9-9a00-47e4-a7d3-9f0db653de31)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/119398292/943b0542-48ea-432b-97c6-ff22e332d0d6)

**Comment by holyuming** [comment]
include.rtl.list 要加上你的design & bram

**Reply by ghost** [reply]
非常感謝，這個問題有順利解決
